new/LIT6945CommentHebrewWRuf.xml

```diff
@@ -0,0 +1,82 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Explications by Walda Rufāel of Goǧǧām in Amharic on 
+                    Commentary on the Epistle to the Hebrews by John Chrysostom</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BNFabb145"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Explications to the Gǝʿǝz <ref type="work" corresp="#LIT3171Commen">Commentary on 
+                    the Epistle to the Hebrews by John Chrysostom</ref> dictated by <persName ref="PRS14363WaldaRufael">ʾalaqā Walda Rufāʾel of Goǧǧām</persName> 
+                    at <placeName ref="LOC6373Yawes">Yawǝš</placeName>, upon the order of <persName ref="PRS4779GossuZa"/>. According to A. d'Abbadie, 
+                    a copy was made from the <persName ref="PRS4779GossuZa">Goššu's</persName> own manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="AmharicLiterature"/>
+                    <term key="Commentary"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+                <language ident="am">Amharic</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2024-03-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="dcterms:creator" active="LIT6945CommentHebrewWRuf" passive="PRS14363WaldaRufael"/>
+                    <relation name="iha:Commentary_of" active="LIT6945CommentHebrewWRuf" passive="LIT3171Commen"/>
```

Comment:
never use `iha `prefixed keywords, relations, files etc, these are IHA-project specific, we use `saws `(saws:isCommentOn)

Reply:
ok